tests/python_tests/test_tokenizer.py

```diff
@@ -280,86 +283,132 @@ def test_special_tokens(tmp_path, prompt, model_id):
     assert decoded_genai_skip_spec != decoded_genai_no_skip
     assert decoded_hf_skip_spec != decoded_hf_no_skip
 
-@pytest.fixture
-def hf_ov_genai_models(tmp_path_factory):
-
-    @functools.lru_cache(maxsize=2)
-    def _get_model_path(model_id, **args):
-        hf_args = args.copy()  # to overcome mutable default argument side effects
-        if "padding_side" in hf_args and hf_args["padding_side"] is None:
-            # HF does not accept None.
-            # Need to remove padding_side and let HF to choose default value,
-            hf_args.pop("padding_side")
-        else:
-            hf_args["truncation_side"] = hf_args["padding_side"]
-        model_dir = tmp_path_factory.getbasetemp() / model_id.replace('/', '_')
-        model_dir.mkdir(exist_ok=True, parents=True)
-
-        hf_tokenizer = AutoTokenizer.from_pretrained(model_id, **hf_args)
-        convert_and_save_tokenizer(hf_tokenizer, model_dir)
-        genai_tokenzier = Tokenizer(model_dir)
-        return hf_tokenizer, genai_tokenzier
-    return _get_model_path
+
+@pytest.mark.precommit
+@pytest.mark.nightly
+def test_miltiple_infer_request_state():
+    hf_tokenizer = retry_request(lambda: AutoTokenizer.from_pretrained("llamafactory/tiny-random-Llama-3"))
+    ov_tokenizer = convert_tokenizer(hf_tokenizer)
+    with TemporaryDirectory() as temp_dir:
```

Comment:
Use pytest's `tmp_path`. It's not remove right after the run so you can use it for reproduction it if needed

Reply:
Done